app/controllers/chapters_controller.rb

```diff
@@ -281,9 +281,9 @@ def set_instance_variables
 
     @allpseuds = (current_user.pseuds + (@work.authors ||= []) + @work.pseuds + (@chapter.authors ||= []) + (@chapter.pseuds ||= [])).uniq
     @pseuds = current_user.pseuds
-    @coauthors = @allpseuds.select{ |p| p.user.id != current_user.id}
-    to_select = @chapter.authors.blank? ? @chapter.pseuds.blank? ? @work.pseuds : @chapter.pseuds : @chapter.authors
-    @selected_pseuds = to_select.collect {|pseud| pseud.id.to_i }
+    @coauthors = @allpseuds.select{ |p| p.user.id != current_user.id }
+    @to_select = @chapter.authors.blank? ? @chapter.pseuds.blank? ? @work.pseuds : @chapter.pseuds : @chapter.authors
```

Comment:
Yeah, so do I -- it took me hours to figure out what it was doing. However, I'm not too keen on refactoring it here, partially because I'd have to update two or three other pull requests, and partially because the corresponding code in the works controller should also be updated if we change this, and that's getting beyond the scope of this issue.